app/src/main/java/com/aspirebudgetingmobile/aspirebudgeting/activities/Login.java

```diff
@@ -0,0 +1,103 @@
+package com.aspirebudgetingmobile.aspirebudgeting.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.RelativeLayout;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+
+import com.aspirebudgetingmobile.aspirebudgeting.R;
+import com.google.android.gms.auth.api.signin.GoogleSignIn;
+import com.google.android.gms.auth.api.signin.GoogleSignInAccount;
+import com.google.android.gms.auth.api.signin.GoogleSignInClient;
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions;
+import com.google.android.gms.common.api.ApiException;
+import com.google.android.gms.common.api.Scope;
+import com.google.android.gms.tasks.Task;
+import com.google.android.material.card.MaterialCardView;
+
+public class Login extends AppCompatActivity {
```

Comment:
Is this Login class a view that the user will see? If yes, can you abstract the interaction with the Google Sign In sdk away from this class and put it in it's own class? Refer to Facilities/UserManager.swift in iOS